Makefile

```diff
@@ -3,10 +3,12 @@ DEB_PATH = "iguazio_deb"
 BINARY_NAME = "igz-fuse"
 RELEASE_VERSION = "0.6.1"
 DOCKER_HUB_USER = "iguaziodocker"
+QUAY_IO_USER = "iguazio"
```

Comment:
Isn't this user should be `quay.io/iguazio`?

Reply:
no need for the ?  mark